client_test/directfund_integration_test.go

```diff
@@ -41,9 +44,29 @@ func TestDirectFundIntegration(t *testing.T) {
 	chain := chainservice.NewMockChain()
 	broker := messageservice.NewBroker()
 
-	clientA := setupClient(alice.PrivateKey, chain, broker, logDestination, 0)
-	clientB := setupClient(bob.PrivateKey, chain, broker, logDestination, 0)
+	clientA, storeA := setupClient(alice.PrivateKey, chain, broker, logDestination, 0)
+	clientB, storeB := setupClient(bob.PrivateKey, chain, broker, logDestination, 0)
 
 	directlyFundALedgerChannel(t, clientA, clientB)
 
+	want := testdata.Outcomes.Create(*clientA.Address, *clientB.Address, 5, 5)
+	// Ensure that we create a consensus channel in the store
+	for _, store := range []store.Store{storeA, storeB} {
+		con, ok := store.GetConsensusChannel(*clientA.Address, *clientB.Address)
+
```

Comment:
Could you assert on the type of `con` to catch the leader/follower typo you made above?

Although, given #462 those types may not exist for long.